simplq/src/styles/statusPage.module.scss

```diff
@@ -103,17 +103,79 @@
   }
 }
 
+
 .token-number {
   z-index: 10;
-  padding: 5%;
-  p {
+  width:225px;
+  span {
     @include center-horizontally();
     @include text-color();
-    font-size: 1rem;
+    // font-size: 1rem;
```

Comment:
remove

Reply:
removed comment. It should be font-size: 1 rem; now